README.markdown

```diff
@@ -196,6 +196,51 @@ You can check the trasaction [status](https://api-reference.datatrans.ch/#tag/v1
   end
 ```
 
+Merchant Initiated Payments
+---------
+
+It's possible to authorize transactions without user interaction, via [merchant initiated payments](https://docs.datatrans.ch/docs/merchant-initiated-payments).
+
+To perform a so-called "dedicated registration" (so we can later charge the card via its `alias`), you should follow the same steps as described above, but not provide an amount:
```

Comment:
There appears to be a second method to issue merchant-initiated payments:

https://docs.datatrans.ch/docs/customer-initiated-payments

> Your customer pays and the card or payment method information is registered. You receive an alias which you save for later merchant initiated payments or one-click checkouts.

I would like to document this as well, but the documentation by Datatrans is really sparse - it is documented that you can do this, but not really how.

Do you have more details on this?